me.gladwell.eclipse.m2e.android/src/main/java/me/gladwell/eclipse/m2e/android/AndroidMavenProjectConfigurator.java

```diff
@@ -71,7 +71,7 @@ public void configure(ProjectConfigurationRequest request, IProgressMonitor moni
                     .getProject());
 
             if (mavenProject.isAndroidProject()) {
-                if (shouldCallJavaProjectConfigurator()) {
+                if (!usesM2E1_6OrNewer()) {
```

Comment:
Improved descriptive method name, but I think its better to have a method that returns whether we're using M2E 1.5 or earlier instead to avoid the use of the negation `!` operators which can be confusing.